src/lib/browse/layers/CrossingsLayerControl.svelte

```diff
@@ -0,0 +1,64 @@
+<script lang="ts">
+  import { circleRadius } from "colors";
+  import { ExternalLink } from "lib/common";
+  import type { MapGeoJSONFeature } from "maplibre-gl";
+  import crossingsUrl from "../../../../assets/crossings.geojson?url";
+  import PointAmenityLayerControl from "./PointAmenityLayerControl.svelte";
+
+  const name = "crossings";
+
+  function tooltip(feature: MapGeoJSONFeature): string {
+    const crossingTypeString: string = feature.properties["crossing"];
```

Comment:
Does type inference not figure out `string` here? Also you can do `properties.crossing` to be more consistent with other code